docs/installation/configuration-file/_index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Configuration File"

```

Comment:
The title/location of this page (and wording throughout) probably needs clarification that it is EXPLICITLY about configuring a robot that is NOT cloud connected. We want people to use the cloud by default. So wording all this as a special case is important I think.

Reply:
Right on. Moved copy up top prominently focusing on cloud conf in app, and clarified value gain by app usage.